### DISCUSSION TITLE
Use all dynamic partitions in downstream job/asset

### DISCUSSION THREAD

**Discussion by adjit** [dagster-io/dagster#19835] [Q&A] [answered]
Hi, as of now I have a monolithic asset, that runs every 30 minutes and takes 30 minutes to run. This asset gets all of our device information and stores it as an asset. Based on this article by Sandy - [Partitioned Data Pipelines](https://dagster.io/blog/partitioned-data-pipelines), we are trying to figure out the best way to accomplish this. Since our devices are dynamic (ie growing and shrinking) we were thinking to have a dynamically partitioned asset which we can create/destroy partitions based on devices. 

How can we have our downstream jobs/assets just utilize all of those dynamic partitions?

**Comment by yuhan** [comment, marked as answer]
This blog post may help: https://dagster.io/blog/dynamic-partitioning

> How can we have our downstream jobs/assets just utilize all of those dynamic partitions?

Here's a relevant discussion: https://github.com/dagster-io/dagster/discussions/15375
